components/filters.tsx

```
@@ -187,115 +371,198 @@ export function Paginate({
+      {/* Page info */}
+      <div className="text-center">
+        <p className="text-sm text-gray-600 dark:text-gray-400">
+          Page <span className="font-semibold text-primary-600 dark:text-primary-400">{initialPage}</span> of{" "}
+          <span className="font-semibold text-primary-600 dark:text-primary-400">{total}</span>
+        </p>
+      </div>
+
+      {/* Enhanced Pagination */}
+      <div className="relative">
+        {/* Background glow effect */}
+        <div className="absolute inset-0 bg-gradient-to-r from-primary-500/10 via-primary-600/5 to-primary-500/10 rounded-2xl blur-xl opacity-60"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)